ims/database/db_connection.py

```diff
@@ -20,7 +28,7 @@ class DatabaseConnection:
     # sample_bmi.db should be changed to something more realistic
     # NullPool pool class is equivalent to no connection pool
     # Should be adapted to postgres SQL
-    engine = create_engine('sqlite:///' + _cfg.db_url,
+    engine = create_engine('sqlite:///' + _cfg.db[constants.DB_URL_KEY],
```

Comment:
I may have missed something here but what will happen, if constants.DB_URL_KEY doesn't exist? Do, we have some validation? (This is same as 1st validation that I told in previous  comments.)